spec/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Travis not using 18.04 yet
```

Comment:
what does this give us?

Reply:
Plain copypasta from bolt, as these dockerfiles have not been changed

Reply:
Good point highlighting this, forgot to remove this dockerfile